crates/tests-e2e/tests/close_position.rs

```diff
@@ -24,4 +25,24 @@ async fn can_collab_close_position() {
     wait_until!(test.app.rx.position().unwrap().position_state == PositionState::Closing);
 
     // TODO: Assert that the position is closed in the app and the coordinator
+
+    tracing::info!("Position closed");
+
+    // Ensure we sync the wallet info after funding
+    spawn_blocking(move || api::refresh_wallet_info().expect("to succeed"))
+        .await
+        .unwrap();
+
+    let balance_at_closing = test.app.rx.wallet_info().unwrap().balances.on_chain;
+
+    let invoice =
+        spawn_blocking(move || api::create_invoice_with_amount(50_000).expect("to succeed"))
```

Comment:
I believe the issue is that we initiate the channel with 100k sats, 50k sats go to the app and 50k sats remain at the coordinator. 
This means, you can only receive `50k sats - channel_reserve`.

Reply:
this was it! probably `no_route` was the result of exceeding the channel capacity.